.github/workflows/release.yml

```diff
@@ -42,15 +42,18 @@ jobs:
       - name: Publish artifacts for all Scala versions
         env:
           BINTRAY_USER: ${{ secrets.BINTRAY_USER }}
-          BINTRAY_PASS: ${{ secrets.BINTRAY_PASS }}
+          # note: although the sbt-bintray plugin expects a var name BINTRAY_PASS, we must pass a bintray key
+          BINTRAY_PASS: ${{ secrets.BINTRAY_KEY }}
         run: sbt "+publish"
 
       - name: Publish API and reference documentation
         env:
           GUSTAV_KEY: ${{ secrets.GUSTAV_KEY }}
+          GUSTAV_PASSPHRASE: ${{ secrets.GUSTAV_PASSPHRASE }}
         run: |+
           eval "$(ssh-agent -s)"
           echo $GUSTAV_KEY | base64 -di > .github/id_rsa
           chmod 600 .github/id_rsa
+          ssh-keygen -p -P "$GUSTAV_PASSPHRASE" -N "" -f .github/id_rsa
```

Comment:
In travis we were uploading the private key and then using a passphrase to open it. So, I'm reproducing the same here except that the key is now stored in a secret. 

It would be better to generate a new key since the previous were published here. The passphase was never compromised tough. Still, better safe than sorry.